### DISCUSSION TITLE
web/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#807] [Announcements]
# web/

介绍 以下是前端组开发，负责项目的简单的介绍 项目介绍 主站 JQuery + layui一把梭，后续同学开发主站主要参考layui 主站主站 海外站 C端 textures站点 3dmodels站点 user站点 service站点 news站点 使用NextJS@13作为主要技术框架，使用git的submodule作为子模块复用到各个站复用，后面会...

http://localhost:8080/web/

**Comment by Bamboo742** [comment]
123123
